litebird_sim/__init__.py

```diff
@@ -9,19 +11,34 @@
     write_healpix_map_to_hdu,
     write_healpix_map_to_file,
 )
+from .mpi import HAVE_MPI4PY, MPI_RANK, MPI_SIZE, MPI_COMM_WORLD, MPI_CONFIGURATION
 from .observations import Observation
 from .simulations import Simulation
 
 __author__ = "The LiteBIRD simulation team"
 __version__ = "0.1.0"
 __all__ = [
```

Comment:
I'm not sure it is a good idea....but what do you think about delegating the definition of `__all__`  ot the modules and do simply `from module import *` in `__init__.py`?

Reply:
From what I understand, `__all__` is used by Python whenever it finds the statement `from <library> import *`, and by code linters that check imports. As described in this [Stack Overflow question](https://stackoverflow.com/questions/31079047/python-pep8-class-in-init-imported-but-not-used), you really need to put `__all__` in the module, not in the sub-modules.